python/jupytercad_lab/jupytercad_lab/notebook/cad_document.py

```diff
@@ -64,6 +66,49 @@ def objects(self) -> List[str]:
             return [x["name"] for x in self._objects_array]
         return []
 
+    @classmethod
+    def import_from_file(cls, path: str) -> CadDocument:
```

Comment:
```suggestion
    def import_from(cls, path: str | pathlib.Path) -> CadDocument:
```

It would be more convenient for users if we could support both path types

Reply:
Good thought, added in https://github.com/jupytercad/JupyterCAD/pull/772/commits/86473fe69efc5e78862ee8c491bae78adba39e97